ezidapp/management/commands/proc-datacite.py

```diff
@@ -17,7 +17,7 @@
 
 class Command(ezidapp.management.commands.proc_base.AsyncProcessingCommand):
     help = __doc__
-    display = 'DataCite'
```

Comment:
Was `display` used anywhere else?

Reply:
No -- it was left over from some earlier formatting of log messages.